dataset/dataset.py

```diff
@@ -39,6 +19,11 @@ class AuxTables(Enum):
     inf_values_idx = 6
     inf_values_dom = 7
 
+class CellStatus(Enum):
+    not_set        = 0
```

Comment:
It is better to have the values capitalized

Reply:
I agree 👍